base/setup/lib/setuplib.h

```diff
@@ -167,6 +167,16 @@ ERROR_NUMBER
 LoadSetupInf(
     IN OUT PUSETUP_DATA pSetupData);
 
+#define ERROR_SYSTEM_PARTITION_NOT_FOUND    (ERROR_LAST_ERROR_CODE + 1)
+
+BOOLEAN
+InitSystemPartition(
+    /**/_In_ PPARTLIST PartitionList,       /* HACK HACK! */
+    /**/_In_ PPARTENTRY InstallPartition,   /* HACK HACK! */
+    /**/_Out_ PPARTENTRY* pSystemPartition, /* HACK HACK! */
```

Comment:
Why the /**/ in the beginning?

Reply:
I just used the same formatting as for other functions here (see `UpdateRegistry()`), where for the time being these parameters are needed, but that I plan to remove their usage _**somehow**_ (by e.g. transmitting them in a different way, or removing code that currently need them but shouldn't) in the future.